modules/light-clients/08-wasm/keeper/keeper_test.go

```diff
@@ -5,23 +5,35 @@ import (
 	"errors"
 	"testing"
 
+	wasmvm "github.com/CosmWasm/wasmvm"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
 	dbm "github.com/cosmos/cosmos-db"
 	testifysuite "github.com/stretchr/testify/suite"
 
 	"cosmossdk.io/log"
+	storetypes "cosmossdk.io/store/types"
 
 	"github.com/cosmos/cosmos-sdk/baseapp"
 	"github.com/cosmos/cosmos-sdk/runtime"
 	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/keeper"
 	wasmtesting "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing/simapp"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
+const (
+	defaultWasmClientID = "08-wasm-0"
```

Comment:
just a nit but could we move this to a general value? we also declare it in `types_test`

Reply:
We cannot right? As test packages are not exported no matter where they are. So we wouldn't be able to export this constant unless we made it a part of our public package.

Reply:
yea or move it to `08-wasm/testing`. I'm happy with whatever, doesn't feel too much of a issue to me to define once per package

Reply:
Yeah, I think it makes more sense to move all of mockvm logic there, and if we end up doing that refactor, then we can move the constant along with it.